src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/NPathComplexityCheck.java

```diff
@@ -102,10 +97,10 @@ public void setMax(int max) {
             TokenTypes.LITERAL_IF,
             TokenTypes.LITERAL_ELSE,
             TokenTypes.LITERAL_SWITCH,
-            TokenTypes.LITERAL_CASE,
-            TokenTypes.LITERAL_TRY,
-            TokenTypes.LITERAL_CATCH,
```

Comment:
Possible calculations:
http://stackoverflow.com/questions/31755570/calculate-npath-complexity-for-try-catch-finally?rq=1 (unanswered)
https://github.com/pdepend/pdepend/blob/master/src/main/php/PDepend/Metrics/Analyzer/NPathComplexityAnalyzer.php#L469-L509 (php way)